src/components/SystemForActionButton.js

```diff
@@ -60,8 +60,11 @@ const SystemForActionButton = ({ issue, remediation, onDelete }) => {
     };
 
     useEffect(() => {
-        loadInventory();
-    }, []);
+        if (open) {
```

Comment:
I don't think this is the proper way, this will eventually create new instance of inventory everytime user openes/closes modal.